README.md

```diff
@@ -148,6 +148,35 @@ Yes, you can! 💡
 ### What is the final result?
    The final result should be similar to what you see in the screenshot below. Notice the logs path. Also, notice the the nested lab vCenter IP address (i.e., 10.1.1.2), because GroupNumber was set to 1 and NumberOfNestedLabs was set to 1. ![screenshot](images/final-screenshot.png)
 
+## No System Assigned Managed Identity
+
+In case you cannot deploy a System Assigned Managed Identity on the Jump server used to deploy resources, follow the follwing process:
+
+   1) From Jumpbox VM, open Command Prompt (cmd.exe).
+   2) Change directory to C:\Temp by running:
+```powershell
+cd c:\Temp\
+```
+   3) Create file `C:\Temp\nestedlabs.yaml` with the following content and replace the values with the ones matching your environment:
+
+```yaml
+AVSvCenter:
+  URL: "X.Y.Z.2" # Please enter the URL for AVS vCenter, do not include https:// or any slashes
```

Comment:
You probably meant, IP not URL, and IP not Host for the yaml example, based on the logic below.

Reply:
Absolutely right.